external/sql/README.md

```diff
@@ -70,17 +70,31 @@ To run this example, users need to include the data sources (`storm-sql-kafka` i
 class path. One approach is to put the required jars into the `extlib` directory:
 
 ```
-$ cp curator-client-2.5.0.jar curator-framework-2.5.0.jar zookeeper-3.4.6.jar
- extlib/
-$ cp scala-library-2.10.4.jar kafka-clients-0.8.2.1.jar kafka_2.10-0.8.2.1.jar metrics-core-2.2.0.jar extlib/
-$ cp json-simple-1.1.1.jar extlib/
-$ cp storm-kafka-*.jar storm-sql-kafka-*.jar storm-sql-runtime-*.jar extlib/
+calcite-avatica-1.4.0-incubating.jar
```

Comment:
Unfortunately you need to find them from maven central and add them to extlib. Note that they'll be added to worker classpath for all workers on cluster.

I know it's really bad, so I'm thinking about how to resolve this.
For now I'm working on adding dependencies from submission step. 
That would be similar (nearly same) to --jars and --packages for spark-submit.